redaxo/src/addons/mediapool/lib/service_media.php

```diff
@@ -229,7 +229,7 @@ public static function updateMedia(array $data, $userlogin = null): array
     public static function deleteMedia(string $filename):bool
     {
         if ($uses = rex_mediapool::mediaIsInUse($filename)) {
-            throw new rex_api_exception(rex_i18n::msg('pool_file_delete_error', $filename) . ' ' . rex_i18n::msg('pool_object_in_use_by', $uses));
+            throw new rex_api_exception(rex_i18n::msg('pool_file_delete_error', $filename) . ' ' . rex_i18n::rawMsg('pool_object_in_use_by', $uses));
```

Comment:
falls das ein bug ist, der schon vorhanden war und nichts mit dem "einführen" der neuen klasse zu tun hat, bitte nicht in diesem PR, sondern in separatem hinterher.

Reply:
vermutlich braucht er jetzt rawMsg, weil jetzt die $uses mit an die message übergeben werden (siehe änderungen in den sprachdateien), und vermutlich enthält $uses HTML.
Wobei ich mich frage, wieso wir es dann jetzt mit in die message ziehen. gerade wenn es html ist, würde ich es tendenziell draußen lassen und msg statt rawMsg nutzen.

Reply:
Das kann ich nicht so leicht voneinander trennen. Die neue Methode wirft die Fehler und gibt sie daher anders zurück und daher muss es bereits hier mit rein und nicht wie vorher, erst später. Im Gesamtkontext sollte es keinen Unterschied machen, im Gegenteil.